packages/xod-arduino/src/templates.js

```diff
@@ -97,6 +97,8 @@ const cppType = def(
   )
 );
 
+const escapeCppString = str => R.replace(/"/g, '\\"', unquote(str));
```

Comment:
It's time to make escaping safer.
Here is the case that fails in compile-time, but the same tweak value then works fine in live session: `"mwa\\ha"ha\"` (first and last quotes is a part of literal)